src/test/ui/consts/const-int-sign.rs

```diff
@@ -3,4 +3,7 @@ fn main() {
     //~^ ERROR temporary value dropped while borrowed
     let y: &'static bool = &(5_i32.is_positive());
     //~^ ERROR temporary value dropped while borrowed
+
+    let sgn: &'static i32 = &(5_i32.signum());
+    //~^ ERROR temporary value dropped while borrowed
```

Comment:
I think this file is supposed to test that results of calls to const fns aren't promoted? Do I need to add to this?

Reply:
not really, you can remove it again if you want

Reply:
They're gone now. I'll ping you when this passes CI.